### PR TITLE
Add link to examples repository

### PR DESCRIPTION
**Pull request by jamwaffles**
A carryon from #17 where all examples where moved to embedded-graphics/examples.

### CODE REVIEW

**Comment by rfuest** [comment]
Should we merge the two `Examples` sections in the README?

**Comment by jamwaffles** [comment]
Well spotted... I was moving a bit too fast. I've merged both sections.

File: src/lib.rs [hunk 1]
```diff
@@ -4,6 +4,10 @@
 //! [embedded-graphics](https://crates.io/crates/embedded-graphics) code, or produce examples and
 //! interactive demos to show of embedded graphics features.
 //!
+//! # [Examples](https://github.com/embedded-graphics/examples)
+//!
+//! Code using the simulator can be found in the [examples repository](https://github.com/embedded-graphics/examples).
+//!
```
Comment (rfuest) [inline src/lib.rs]: I believe this should have been removed.

**Review by rfuest — CHANGES_REQUESTED** [review]
Should we move the examples section above the setup section? IMO the examples are more relevant for users that look at the README for the first time.

File: src/lib.rs (hunk 1 above)
Comment (jamwaffles) [reply src/lib.rs]: Thanks for catching that. Fixed now, along with reorganising the sections and a typo fix.

**Review by rfuest — APPROVED** [review]
LGTM

**Comment by jamwaffles** [comment]
Thanks!

**Pull request merged**